tensorflow/lite/kernels/BUILD

```diff
@@ -539,6 +540,7 @@ cc_library(
         "//tensorflow/lite/kernels/internal:audio_utils",
         "//tensorflow/lite/kernels/internal:common",
         "//tensorflow/lite/kernels/internal:compatibility",
+        "//tensorflow/lite/kernels/internal:cppmath",
```

Comment:
Hmm, shouldn't you be adding the dependency to lite/kernels/internal:common? Rather than indirect downstream dependencies?

Reply:
yes, you're right.
done